source/guides/puppetlabs_package_verification.markdown

```diff
@@ -35,31 +35,32 @@ Before you can verify signatures, you must import the Puppet Labs public key and
 
 To import the release signing key, run:
 
-    $ gpg --recv-key 1054b7a24bd6ec30
-    gpg: requesting key 4BD6EC30 from hkp server pool.sks-keyservers.net
-    gpg: key 4BD6EC30: public key "Puppet Labs Release Key " imported
+    $ gpg --recv-key 7F438280EF8D349F
+    gpg: requesting key EF8D349F from hkp server pool.sks-keyservers.net
+    gpg: key EF8D349F: public key "Puppet, Inc. Release Key (Puppet, Inc. Release Key) <release@puppet.com>" imported
```

Comment:
Should we leave a note about this tool possibly printing the short key ID instead of the long one regardless of how it's fetched?